### PR TITLE
feat(variables): show status on list variables table

### PR DESCRIPTION
**Pull request by thoughtassassin**
## Related Issue

Add Status To List Variables Table enhancement

## Motivation and Context

To show the user what the status of the variable is

## How Has This Been Tested?

The code was tested locally by calling the actual api.

## Screenshots (if appropriate):
Screen Shot 2021-04-20 at 4 04 43 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: test/commands/environment/set-variables.test.js [hunk 1]
```diff
@@ -255,6 +255,9 @@ test('set-environment-variables - delete secret', async () => {
 })
 
 test('set-environment-variables - delete not found', async () => {
+  const originalWarn = console.warn
```
Comment (justinedelson) [inline test/commands/environment/set-variables.test.js:258]: This change does not seem related to the PR. I'm not sure why this is necessary, but please create a separate issue/PR for this.

**Comment by justinedelson** [comment]
@thoughtassassin This looks OK, other than the extraneous change to the tests. However, the commit message isn't aligned to the CONTRIBUTING document. This is a new feature and should be prefixed with `feat` (perhaps `feat(variables)`) not `chore`. It would also be good to put an issue reference in the commit comment. This was not clear in the CONTRIBUTING file, but I've tried to address this.

File: test/commands/environment/set-variables.test.js (hunk 1 above)
Comment (thoughtassassin) [reply test/commands/environment/set-variables.test.js:258]: @justinedelson I think the tests are broken honestly without this. I will create a separate pr.

**Comment by thoughtassassin** [comment]
@justinedelson I removed the console warning mock
